design/work_item_board.go

```diff
@@ -0,0 +1,109 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var workItemBoardSingle = JSONSingle(
+	"workItemBoard",
+	`A board for work item types`,
+	workItemBoardData,
+	workItemBoardLinks,
+)
+
+var workItemBoardList = JSONList(
+	"workItemBoard",
+	`List of work item boards`,
+	workItemBoardData,
+	workItemBoardLinks,
+	nil,
+)
+
+var workItemBoardLinks = a.Type("WorkItemBoardLinks", func() {
+	a.Attribute("self", d.String, func() {
+		a.Example("http://api.openshift.io/api/spacetemplates/2d98c73d-6969-4ea6-958a-812c832b6c18/workitemboards")
+	})
+	a.Required("self")
+})
+
+var workItemBoardColumnData = a.Type("WorkItemBoardColumnData", func() {
+	a.Description(`a column represents a vertical lane in a board`)
+	a.Attribute("type", d.String, "The type string of the work item board column", func() {
+		a.Enum("boardcolumns")
+	})
+	a.Attribute("id", d.UUID, "ID of the work item board column", func() {
+		a.Example("712f20e4-2202-4469-9a02-b892b7051b2b")
+	})
+	a.Attribute("attributes", workItemBoardColumnAttributes)
+	a.Required("id", "type", "attributes")
```

Comment:
`version` should also be a required attribute

Reply:
I neither is on `space` nor `labels`, so why here? When I create a board I shouldn't be required to pass in the version.